src/main/java/com/conveyal/r5/streets/EgressCostTable.java

```diff
@@ -269,96 +307,38 @@ public EgressCostTable (LinkedPointSet linkedPointSet, ProgressListener progress
 
                 if (streetMode == StreetMode.BICYCLE) {
                     sr.distanceLimitMeters = linkingDistanceLimitMeters;
-                    sr.quantityToMinimize = linkageCostUnit;
-                    sr.route();
-                    return linkedPointSet.extendDistanceTableToPoints(sr.getReachedVertices(), envelopeAroundStop);
                 } else if (streetMode == StreetMode.CAR) {
-                    // The speeds for Walk and Bicycle can be specified in an analysis request, so it makes sense above to
-                    // store distances and apply the requested speed. In contrast, car speeds vary by link and cannot be
-                    // set in analysis requests, so it makes sense to use seconds directly as the linkage cost.
+                    // Car speeds vary by link and cannot be set in analysis requests, so it makes sense to use
+                    // seconds directly as the linkage cost. In contrast, the speeds for Walk and Bicycle can be
+                    // specified in an analysis request, so it makes sense above to store distances and apply the
+                    // requested speed.
                     // TODO confirm this works as expected when modifications can affect street layer.
                     sr.timeLimitSeconds = CAR_TIME_LINKING_LIMIT_SECONDS;
-                    sr.quantityToMinimize = linkageCostUnit;
-                    sr.route();
-                    // TODO optimization: We probably shouldn't evaluate at every point in this LinkedPointSet in case
-                    //      it's much bigger than the driving radius.
-                    PointSetTimes driveTimesToAllPoints = linkedPointSet.eval(
-                            sr::getTravelTimeToVertex,
-                            null,
-                            LinkedPointSet.OFF_STREET_SPEED_MILLIMETERS_PER_SECOND,
-                            null
-                    );
-                    // TODO optimization: should we make spatial index visit() method public to avoid copying results?
-                    TIntList packedDriveTimes = new TIntArrayList();
-                    for (int p = 0; p < driveTimesToAllPoints.size(); p++) {
-                        int driveTimeToPoint = driveTimesToAllPoints.getTravelTimeToPoint(p);
-                        if (driveTimeToPoint != Integer.MAX_VALUE) {
-                            packedDriveTimes.add(p);
-                            packedDriveTimes.add(driveTimeToPoint);
-                        }
-                    }
```

Comment:
Consolidated in linkedPointSet.extendCostsToPoints (around https://github.com/conveyal/r5/pull/647/files#diff-dd60435552e26291d0b9b9a9ad8bfa02c92db82d5217698bc03f98665b96eebdR537)